### DISCUSSION TITLE
Docker usage

### DISCUSSION THREAD

**Discussion by dupuysylvain** [seriousm4x/UpSnap#177] [Help wanted] [answered]
Hello,

I use this application using the docker image with Unraid (https://unraid.net/)

What path i should persist to update the container without loosing data (actually, i loose everything) ?

Thanks,
Sylvain Dupuy

**Comment by dupuysylvain** [comment, marked as answer]
Update : 

I map the "/app/pb_data" path of docker, it seems to work for now.

**Reply by seriousm4x** [reply]
Yes, thats the correct path
